blaze-core/src/main/scala/org/http4s/blazecore/websocket/Serializer.scala

```diff
@@ -0,0 +1,150 @@
+package org.http4s.blazecore.websocket
+
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.{AtomicInteger, AtomicReference}
+
+import org.http4s.blaze.pipeline.TailStage
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Future, Promise}
+import scala.collection.mutable.ArrayBuffer
+import org.http4s.blaze.util.Execution._
+
+import scala.util.{Failure, Success, Try}
```

Comment:
We're generally sorting the imports alphabetically.  We got the idea from your employer. :laughing: